tools/cli/src/main.rs

```diff
@@ -108,6 +103,15 @@ fn aqua(args: &clap::ArgMatches<'_>) -> Result<(), anyhow::Error> {
         }
     };
     let service_name = service_name.to_pascal_case();
+
+    // this line allows exporting all the stuff from generated aqua module definition
+    println!("module {} declares *\n", service_name);
```

Comment:
Add one more line break for better readability?

Reply:
Yep, for greeting it'd be
```
module Greeting declares *

service Greeting:
  greeting(name: string) -> string
```